commit ac9609c8f270e2e96ee477346c2b3eea4ae63a7e
Author: Linh Hua <linh.hua.jc@renesas.com>
Date:   Thu Jul 17 17:24:27 2025 +0700

    meta-rz-common: glib2.0: update to 2.58.3-2+deb10u8
    
    Switch to ELTS mirror since deb10u5 is no longer available.
    
    Signed-off-by: Linh Hua <linh.hua.jc@renesas.com>

diff --git a/meta-rz-common/recipes-debian/buster/sources/glib2.0.inc b/meta-rz-common/recipes-debian/buster/sources/glib2.0.inc
index deb3c87b..53a159f2 100644
--- a/meta-rz-common/recipes-debian/buster/sources/glib2.0.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/glib2.0.inc
@@ -1,19 +1,22 @@
 # This is generated by debian-source.bbclass
-DPV = "2.58.3-2+deb10u5"
+DPV = "2.58.3-2+deb10u8"
 DPV_EPOCH = ""
 REPACK_PV = "2.58.3"
 PV = "2.58.3"
 
+# ELTS (Extended LTS) mirror for packages with long-term maintenance
+DEBIAN_ELTS_MIRROR ?= "https://deb.freexian.com/extended-lts/pool"
+
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/glib2.0/glib2.0_2.58.3-2+deb10u5.dsc;name=glib2.0_2.58.3-2+deb10u5.dsc \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/glib2.0/glib2.0_2.58.3.orig.tar.xz;name=glib2.0_2.58.3.orig.tar.xz \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/glib2.0/glib2.0_2.58.3-2+deb10u5.debian.tar.xz;name=glib2.0_2.58.3-2+deb10u5.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/g/glib2.0/glib2.0_2.58.3-2+deb10u8.dsc;name=glib2.0_2.58.3-2+deb10u8.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/g/glib2.0/glib2.0_2.58.3.orig.tar.xz;name=glib2.0_2.58.3.orig.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/g/glib2.0/glib2.0_2.58.3-2+deb10u8.debian.tar.xz;name=glib2.0_2.58.3-2+deb10u8.debian.tar.xz \
 "
 
-SRC_URI[glib2.0_2.58.3-2+deb10u5.dsc.md5sum] = "c9c7fe5ccab432f00a2e92c73620ea68"
+SRC_URI[glib2.0_2.58.3-2+deb10u8.dsc.md5sum] = "404f68f0da63d12b467e7b70d694d241"
 SRC_URI[glib2.0_2.58.3.orig.tar.xz.md5sum] = "8058c7bde846dcffe5fa453eca366d73"
-SRC_URI[glib2.0_2.58.3-2+deb10u5.debian.tar.xz.md5sum] = "e8ad6cbb292b5a3662ff4d03046c968c"
+SRC_URI[glib2.0_2.58.3-2+deb10u8.debian.tar.xz.md5sum] = "e30d4f2019491327facb09402a4eb81f"
 
-SRC_URI[glib2.0_2.58.3-2+deb10u5.dsc.sha256sum] = "59c25e933d20f4f711c4b6685fbfeb46f1df344aa0e09c862b77fcaef94c57d3"
+SRC_URI[glib2.0_2.58.3-2+deb10u8.dsc.sha256sum] = "08ab8c077f196916fa4d8ceb889fe035f558422948b627deed86d3e4a75af61b"
 SRC_URI[glib2.0_2.58.3.orig.tar.xz.sha256sum] = "8f43c31767e88a25da72b52a40f3301fefc49a665b56dc10ee7cc9565cbe7481"
-SRC_URI[glib2.0_2.58.3-2+deb10u5.debian.tar.xz.sha256sum] = "5c9d1dc438c0a8923eaf65391e04906230d06a71426dc1aa6df785b12a4d21d4"
+SRC_URI[glib2.0_2.58.3-2+deb10u8.debian.tar.xz.sha256sum] = "7bf28497a3ed2ac82d471d5bd1fcd29367a57c57e15c2fe86f4902f834920eeb"
